docs/reference/platforms/linux/appimage.rst

```diff
@@ -41,6 +41,43 @@ AppImage:
    using Ubuntu 19.10, for example, you can expect that only people on the most
    recent versions of another distribution will be able to run your AppImage.
 
+Build Process
+=============
```

Comment:
It's definitely worth documenting this from a "how it works" perspective, but I wonder about the specific placement here. The information that will be more directly applicable are the details about Docker usage, Icon naming, and so on; this block is more in the camp of "something has gone wrong/this is what we're doing so that you can debug" territory. I think it might be better served by being at the *end* of this document, as part of an "Implementation details" section.

Reply:
Great suggestion, thanks! I moved the section to the end of the document.